.env.template

```diff
@@ -1,2 +1,2 @@
 APP_ID=
-OFFLINE_LICENSE_TOKEN=
+ONLINE_AUTH_TOKEN=
```

Comment:
Is this an accident?

Reply:
The code requests the Online auth token and not the offline license token, so I changed the .env file to match and to stay inline with what we have for Android inventory

Reply:
Thanks!